uber/models.py

```diff
@@ -1860,3 +1880,36 @@ def register_session_listeners():
     listen(Session.session_factory, 'after_flush', _release_badge_lock)
     listen(Session.engine, 'dbapi_error', _release_badge_lock_on_error)
 register_session_listeners()
+
+
+"""
+Initialize the database on startup
+
+We want to do this only after all other plugins have had a chance to initialize
+and add their 'mixin' data (i.e. extra colums) into the models.
+
+Also, it's possible that the DB is still initializing and isn't ready to accept connections, so,
+if this fails, keep trying until we're able to connect.
+
+This should be the ONLY spot (except for maintenance tools) in all of core ubersystem or any plugins
+that attempts to create tables by passing modify_tables=True to Session.initialize_db()
+"""
+
+
+def initialize_db():
+    num_tries_remaining = 10
+    while not stopped.is_set():
+        try:
+            Session.initialize_db(modify_tables=True)
+        except KeyboardInterrupt:
```

Comment:
This is real nit-picky, but we use single quotes rather than double quotes when possible.  (We only use double quotes when the string itself contains single quotes.)  I'll see if I can get PEP8 to check for that.